src/frontend/screens/Settings/sections/SystemInfo/cpu.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Grid from '@mui/material/Grid'
+import Paper from '@mui/material/Paper'
+import Typography from '@mui/material/Typography'
+
+import type { SystemInformation } from 'backend/utils/systeminfo'
+import { useTranslation } from 'react-i18next'
+import VendorLogo from './vendorLogo'
+
+function CPUCard({ cpu }: { cpu: SystemInformation['CPU'] }) {
+  const { model, cores } = cpu
+  const { t } = useTranslation()
+
+  return (
+    <Paper sx={{ padding: 1, height: '100%' }} square>
+      <Typography variant="h6">
+        {t('settings.systemInformation.CPU', 'CPU:')}
```

Comment:
```suggestion
        {t('settings.systemInformation.cpu', 'CPU:')}
```

for consistency with the other keys (gpu, os)